app/livechat/server/api/lib/users.js

```diff
@@ -0,0 +1,55 @@
+import { hasPermissionAsync } from '../../../../authorization/server/functions/hasPermission';
+import { Roles } from '../../../../models/server/raw';
+
+async function findUsers({ userId, role, pagination: { offset, count, sort } }) {
+	if (!await hasPermissionAsync(userId, 'view-livechat-manager') || !await hasPermissionAsync(userId, 'manage-livechat-agents')) {
+		throw new Error('error-not-authorized');
+	}
+
+	const cursor = await Roles.findUsersInRoles(role, undefined, {
+		sort: sort || { name: 1 },
+		skip: offset,
+		limit: count,
+	});
+
+	const total = await cursor.count();
+
+	const users = await cursor.toArray();
+
+	return {
+		users: users.map((user) => Object.assign({}, {
```

Comment:
you can pass to `findUsersInRoles` the list of fields you want, then you don't need this `map`